.cicd/generate-pipeline.sh

```diff
@@ -132,7 +132,7 @@ EOF
           failover-registries:
             - 'registry_1'
             - 'registry_2'
-          pre-execute-sleep: 10
+          pre-execute-ping-sleep: "8.8.8.8"
```

Comment:
what's this change all about?

Reply:
The current 10 second sleep isn't adequate, so the new ping sleep waits until it can reach 8.8.8.8 and then allows commands to proceed. This avoids having to change from 10 every so often because it's not enough to prevent networking failures.